internal/pkg/addons/testdata/merge/first.yaml

```diff
@@ -26,3 +26,5 @@ Mappings:
 
 Conditions:
     IsProd: !Equals [!Ref Env, prod]
+
+Transform: AWS::Serverless-2016-10-31
```

Comment:
why is this considered as a valid format?

Reply:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/transform-section-structure.html

`Transform` is either a single value or a list :P